config/routes.rb

```diff
@@ -46,6 +46,7 @@
 
   get 'pages/home'
   get 'pages/dashboard'
+  get 'pages/services'
```

Comment:
I think I moved away from this - since we already have a services route and so that would be confusing to set that name up for views as well. 

```suggestion
  get 'pages/services'
```